packages/renderer/src/lib/ContainerDetailsLogs.svelte

```diff
@@ -90,14 +93,35 @@ async function refreshTerminal() {
   // call fit addon each time we resize the window
   window.addEventListener('resize', () => {
     if (currentRouterPath === `/containers/${container.id}/logs`) {
-      fitAddon.fit();
+      termFit.fit();
     }
   });
-  fitAddon.fit();
+  termFit.fit();
 }
+
+function resizeTerminal() {
+  if (logsTerminal) {
+    termFit.fit();
+  }
+}
+
 onMount(async () => {
   refreshTerminal();
 });
+
+onMount(() => {
```

Comment:
Is is okay having two `onMount` in svelte? Unsure the most ideal way to implement this. @benoitf

Reply:
better to use once 👍  (especially the first one being really short)